test/ignored-changes.json

```diff
@@ -50,6 +50,10 @@
         {
           "code": "java.method.removed",
           "old": "method io.camunda.zeebe.broker.Broker io.camunda.zeebe.test.EmbeddedBrokerRule::getBroker()"
+        },
+        {
+          "code": "java.class.externalClassNoLongerExposedInAPI",
+          "justification": "The changes above caused us to no longer expose certain types in the API, however we do not want to expose these types, as such ignore these for this release"
```

Comment:
Can we be more explicit about these types? For example, by specifying a specific package or even explicitly declaring these classnames?

Reply:
Yes we can. There were quite a few of them this time, so I made a call in this case since we don't expect to be really exposing any external types in this module. In modules where we do, this would be a different case.